src/formatter.ts

```diff
@@ -97,15 +97,19 @@ export default class Formatter implements vsc.DocumentFormattingEditProvider,
     }
 };
 
-const languageMap = {
-    'apex': 'JAVA',
-    'c': 'C',
-    'cpp': 'CPP',
-    'csharp': 'CS',
-    'd': 'D',
-    'java': 'JAVA',
-    'objective-c': 'OC',
-    'pawn': 'PAWN',
-    'pde': 'JAVA',
-    'vala': 'VALA'
-};
+const langOverrides = vsc.workspace.getConfiguration("uncrustify").get("langOverrides", false);
```

Comment:
Why is the default value `false` if `uncrustify.langOverrides` is an object ?
The default value is only used if the key is not found in the settings. If the value is `null`, then this value will be used instead of the default one from the `get()` function. I would either go for `vsc.workspace.getConfiguration('uncrustify').get('langOverrides") || {}` or just `vsc.workspace.getConfiguration('uncrustify').get('langOverrides')` since `uncrustify.langOverrides` is in the `package.json` (and thus will always be found).